src/sbt-test/sbt-cake/simple/src/main/scala/Hello.scala

```diff
@@ -1 +1 @@
-object TerribleFormatting { def foo = "bar" }
+object TerribleFormatting{def foo=   "bar"}
```

Comment:
Adding in an additional comment here feels like adding an unnecessary comment maintenance burden. So I'm with @fommil, let's keep this code as it currently is.